code/modules/food_and_drinks/recipes/recipes_microwave.dm

```diff
@@ -1017,19 +1091,25 @@
 
 /datum/recipe/microwave/wrap
 	reagents = list("soysauce" = 10)
-	items = list(/obj/item/food/snacks/friedegg, /obj/item/food/snacks/grown/cabbage )
+	items = list(
+		/obj/item/food/snacks/friedegg,
+		/obj/item/food/snacks/grown/lettuce
+		)
```

Comment:
```suggestion
	)
```
github may troll you and make this spaced, but all the others got one indent, but this got two